tests/modules/suite2p_registration/test_suite2p_registration_utils.py

```diff
@@ -36,3 +38,29 @@ def test_identify_and_clip_outliers(excess_indices, deltas,
     deltas = np.abs(data[expected_indices] - baseline[expected_indices])
     small_delta = 1.0
     assert np.all(deltas < thresh + small_delta)
+
+
+def test_check_movie_against_raw():
+    """Test that we can recognize an unordered or non flux conserving motion
+    corrected movie as expected.
+    """
+    rng = np.random.default_rng(1234)
+
+    file_loc = Path(__file__)
+    resource_loc = file_loc.parent / 'resources'
+    h5_file_loc = resource_loc / '792757260_test_data.h5'
+    h5_key = 'input_frames'
+
+    with h5py.File(h5_file_loc) as data_file:
+        corr_data = data_file[h5_key][:]
+
+    #  This should pass without a raise.
+    check_movie_against_raw(corr_data=corr_data,
+                            raw_hdf5=h5_file_loc,
+                            h5py_key=h5_key)
+
+    rand_index = rng.integers(0, corr_data.shape[0], size=corr_data.shape[0])
+    with pytest.raises(ValueError):
+        check_movie_against_raw(corr_data=corr_data[rand_index],
+                                raw_hdf5=h5_file_loc,
+                                h5py_key=h5_key)
```

Comment:
Please add a test that you get an error when the two movies are of different shapes